cooperative_lanechange/src/cooperative_lanechange_node.cpp

```diff
@@ -424,7 +433,12 @@ namespace cooperative_lanechange
       p.planner_plugin_name = get_plugin_name();
     }
 
-    return;
+    std::chrono::system_clock::time_point end_time = std::chrono::system_clock::now();  // Planning complete
+
+    auto duration = end_time - start_time;
```

Comment:
Similar comment to above - I don't see `start_time` defined in this scope, will this build?

Reply:
added